### DISCUSSION TITLE
How does `invite_only` for registration works exactly?

### DISCUSSION THREAD

**Discussion by pineapples5972** [plausible/analytics#4057] [Self-Hosted Support] [answered]
I have set the enviroment variable **"DISABLE_REGISTRATION=invite_only"** However I dont get how to make use of this. 
How do I invite people to register? 
- Is there any link?
- I checked all the UI but couldnt find it.
- Is it really for registration or just allowing selected people to view site stats?

To be clear my intention is to make people to whom I sent invite link to register.
Is this feature really made for this or what?

This is really confusing. Please Let me know if yes then how can I sent them invite link so they will be able to register.

**Comment by ruslandoga** [comment, marked as answer]
👋 @pineapples5972 

You can invite people by mail to your website on, for example, https://who.edify.space/stats.edify.space/settings/people (https://who.edify.space is my self-hosted instance, stats.copycat.fun is the site I want to invite people to).

There is an "Invite" button.

<img width="1582" alt="Screenshot 2024-05-02 at 22 16 10" src="https://github.com/plausible/analytics/assets/67764432/c2d13a8c-d73b-425d-ba10-c5cf619ce28f">

> Is it really for registration or just allowing selected people to view site stats?

It's kind of both. The invited user registers and gets permission to view or administer site stats.

**Reply by pineapples5972** [reply]
Ohh so its kinda both invite and register. 
Got it thanks for clarifying.
